src/library/scala/collection/immutable/LazyList.scala

```diff
@@ -189,6 +189,33 @@ import scala.runtime.Statics
   *  }
   *  }}}
   *
+  *  - Building a LazyList with `#::` or `#:::` creates the tail lazily.
+  *  The head, the tail and whether the list is empty or not can be initially unknown.
+  *  They are all evaluated at once, but using any of those constructs, evaluation
```

Comment:
I feel weird saying they're all evaluated at once, since, while the tail is *technically* evaluated, it's *effectively* not given the double laziness boxing.